pkg/authconfigmap/authconfigmap.go

```diff
@@ -1,185 +1,241 @@
+// Package authconfigmap allows manipulation of the EKS configmap,
+// which maps IAM entities to Kubernetes groups.
+// See for more information:
+// - https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html
+// - https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/master/README.md#full-configuration-format
 package authconfigmap
 
 import (
 	"fmt"
+	"sort"
 
 	"github.com/kris-nova/logger"
 	"github.com/pkg/errors"
-
-	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha4"
-
-	yaml "gopkg.in/yaml.v2"
-
+	"gopkg.in/yaml.v2"
```

Comment:
While at it, could you please change this to `sigs.k8s.io/yaml`, it's not the preferred version. I believe the interface is 100% the same.